crm_claim_rma/wizard/claim_make_picking.py

```diff
@@ -0,0 +1,279 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Eezee-It
+#    Copyright 2013 Camptocamp
+#    Copyright 2009-2013 Akretion,
+#    Author: Emmanuel Samyn, Raphaël Valyi, Sébastien Beau,
+#            Benoît Guillot, Joel Grand-Guillaume
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.models import api, TransientModel, _
+from openerp.fields import Many2many, Many2one
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.exceptions import Warning
+from openerp import workflow
+from openerp import workflow
+
+import time
+
+
+class ClaimMakePicking(TransientModel):
+    _name = 'claim_make_picking.wizard'
+    _description = 'Wizard to create pickings from claim lines'
+
+    # Get default source location
+    def _get_source_loc(self):
```

Comment:
@aristobulo-eezee I think that this method should have the @ api.model because it is called from a field of API 8.0  and without @api.model the default not working. Did you do a functional test in a test instance about this method for default of claim_line_source_location field?